pandas/__init__.py

```diff
@@ -58,6 +58,7 @@
 from pandas.io.api import *
 from pandas.util._tester import test
 import pandas.testing
+from . import computation  # TODO: Remove when deprecated shim module is removed
```

Comment:
this will trigger the deprecation warning already (so showing it on just importing pandas), so we have to have a workaround this